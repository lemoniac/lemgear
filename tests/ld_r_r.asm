ld A, 10
ld B, A
ld C, A
ld D, A
ld E, A
ld H, A
ld L, A

halt
