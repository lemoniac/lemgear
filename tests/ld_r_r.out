AF : 0A00
BC : 0A0A
DE : 0A0A
HL : 0A0A
